features/works/work_create.feature

```diff
@@ -197,6 +197,41 @@ Feature: Create Works
       And I should see "Chapter"
       And I should see "1/?"
 
+  Scenario: Creating a new work in a new series with some invalid things should return to the new work page with an error message and series information still filled in
+    Given basic tags
+      And I am logged in as "thorough" with password "something"
+    When I set up the draft "Bad Draft"
+      And I fill in "Fandoms" with "Invalid12./"
+      And I fill in "Work Title" with "/"
+      And I fill in "content" with "T"
+      And I check "This work has multiple chapters"
+      And I fill in "Post to Collections / Challenges" with "collection1, collection2"
+      And I check "This work is part of a series"
+      And I fill in "Or create and use a new one:" with "My new series"
+      And I press "Preview"
+    Then I should see "Sorry! We couldn't save this work because:"
+      And I should see a collection not found message for "collection1"
+      And the field labeled "Or create and use a new one:" should contain "My new series"
+      And I should not see "Remove Work From Series"
+
+  Scenario: Creating a new work in an existing series with some invalid things should return to the new work page with an error message and series information still filled in
+    Given basic tags              |
```

Comment:
Travis contradicts me /o\ There's a stray `|` at the end of this line that is causing an "undefined step" error.